package.json

```diff
@@ -179,7 +179,7 @@
 		"grunt": "grunt",
 		"lint:jsdoc": "wp-scripts lint-js",
 		"lint:jsdoc:fix": "wp-scripts lint-js --fix",
-		"env:start": "node ./tools/local-env/scripts/start.js",
+		"env:start": "node ./tools/local-env/scripts/start.js && node ./tools/local-env/scripts/docker.js run -T php composer update -W",
```

Comment:
Hey, @desrosj wondering if `env:install` is a better option as Composer or updates the dependencies.

Reply:
`env:install` seems fine, but because tests are run within the containers, `composer update` needs to also run in the container in order to match PHP versions.

Once someone runs `env:install`, they won't be re-running that command unless they are recreating their environment from scratch. When this is on `env:start` or `test:php`, `composer update` is run ensuring the latest versions are always installed before running tests.

Reply:
@hellofromtonya It doesn't seem like you strongly object to `env:start`, so I am going to move forward with that for now. We can always consider moving `composer update` to `env:install` in the future!